compiler/il/OMRIL.cpp

```diff
@@ -1,5 +1,9 @@
 /*******************************************************************************
+<<<<<<< HEAD
  * Copyright (c) 2000, 2020 IBM Corp. and others
+=======
```

Comment:
There are extra lines from git rebase operation.

Reply:
Sorry for the mistake. Fixed in the new push.